tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932250.yaml

```diff
@@ -4,7 +4,7 @@ meta:
   description: |
     Tests to trigger or not trigger rule 932250.
     - commands used must be less than 4 chars
-    - [\s<>&|)] is required after a command to reduce FPs
+    - \b is required after a command to reduce FPs
```

Comment:
I think this is no longer true after your last update.

Reply:
Good catch, I'll fix it in a sec